src/components/CVModal/container.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { fetchCV } from 'Services';
+import { getEmployeeCV } from 'Reducers/selected-options';
+
+const CVModalContainer = Wrapped =>
+  class extends React.Component {
+    static propTypes = {
+      employeeCV: PT.shape({ id: PT.number, label: PT.string }).isRequired,
+    };
+
+    state = { cv: null };
+
+    componentDidMount() {
+      fetchCV().then(data => this.setState({ isModalOpen: false, cv: data }));
+    }
+
+    componentDidUpdate(prevProps) {
+      const { employeeCV } = this.props;
+      if (prevProps.employeeCV !== employeeCV) {
+        this.setState({ employeeCV, isModalOpen: true });
+      }
+    }
+
+    handleCancelModal = () => {
```

Comment:
`handleCancelModal = () => this.setState({ isModalOpen: false });`

You can write it up as above. If you're basically returning something straight away. It's a bit cleaner to write it like this. I know you've done it in other places. It's just to make it consistent across the app.